tests/lib/managers/folders-test.js

```diff
@@ -276,4 +276,21 @@ describe('Folders', function() {
 		});
 	});
 
+	describe('getTrashedItems()', function() {
+
+		it('should make GET call to to get trashed items', function() {
```

Comment:
The names of these tests are reversed; the name of the test should indicate which thing is having expectations placed on it with the sinon mock.